sinonjs/sinon
# Replace homegrown support banner with opencollective-cli

This PR replaces the homegrown support banner with `opencollective-cli`

#### How to verify - mandatory
1. Check out this branch
2. `npm run postinstall`
3. Observe banner

![2018-03-26 at 16 22](https://user-images.githubusercontent.com/20321/37912083-e4ce2d02-3111-11e8-8aec-5b770a1d5164.png)
